spec/api-debugger-spec.ts

```diff
@@ -227,20 +227,25 @@ describe('debugger module', () => {
     it('creates unique session id for each target', (done) => {
       w.webContents.loadFile(path.join(__dirname, 'fixtures', 'sub-frames', 'debug-frames.html'));
       w.webContents.debugger.attach();
-      let session: String;
+      let debuggerSessionId: string;
 
       w.webContents.debugger.on('message', (event, ...args) => {
         const [method, params, sessionId] = args;
         if (method === 'Target.targetCreated') {
           w.webContents.debugger.sendCommand('Target.attachToTarget', { targetId: params.targetInfo.targetId, flatten: true }).then(result => {
-            session = result.sessionId;
+            debuggerSessionId = result.sessionId;
             w.webContents.debugger.sendCommand('Debugger.enable', {}, result.sessionId);
+
+            // Ensure debugger finds a script to pause to possibly reduce flaky
+            // tests.
+            w.webContents.mainFrame.executeJavaScript('void 0;');
           });
         }
         if (method === 'Debugger.scriptParsed') {
-          expect(sessionId).to.equal(session);
-          w.webContents.debugger.detach();
-          done();
+          if (sessionId === debuggerSessionId) {
+            w.webContents.debugger.detach();
+            done();
+          }
```

Comment:
This looks like a nice change, but is it intrinsic to the roll? e.g. if this was removed from this PR and put into a separate PR, would that break the roll?

Asking because d37038288a33808c9f07ccf633c2aaf1d1c232a7 looks like a good candidate as a standalone so it can be tropped to older branches

Reply:
good point! I wasn't considering the inability to backport from chromium rolls. I'll wait for this to pass and open a new PR for that CI fix.

Reply:
https://github.com/electron/electron/pull/44741